autowarefoundation/autoware
# [Fix] Resize RTM UI dialog windows to prevent scrolling

## Status
**DEVELOPMENT**

## Description
Resize dialog windows launched from the RTM UI in the Sensing and Computing tabs to prevent scrolling. Implements autowarefoundation/autoware_ai#377 

## Todos
- [x] Testing on HiDPI monitors (optional)

## Steps to Test or Reproduce
Open teh RTM as usual, and open dialog windows in the Sensing and Computing tabs that are available through the [acc] links. No scrolling should be required to see all available parameters.
